diesel/src/pg/types/numeric.rs

```diff
@@ -102,6 +148,7 @@ mod bigdecimal {
 
     impl FromSql<types::Numeric, Pg> for BigDecimal {
         fn from_sql(numeric: Option<&[u8]>) -> deserialize::Result<Self> {
+            // FIXME: Use the TryFrom impl when try_from is stable
             let (sign, weight, scale, digits) = match PgNumeric::from_sql(numeric)? {
```

Comment:
This is maybe a nit, but I feel like in the interim we should pull this into a function so we don't *actually* have 40 lines of code copy-pasted like htis